apps/single-view/src/Views/CustomerView/Notes.tsx

```diff
@@ -14,6 +14,7 @@ import {
 interface Props {
   systemIds?: Array<SystemId>;
   displayNoteInput?: boolean;
+  isHousing: boolean;
```

Comment:
I like this, might be nice to use it [here](https://github.com/LBHackney-IT/single-view-frontend/blob/main/apps/single-view/src/Views/CustomerView/index.tsx#L20) too